client/src/components/ProductDetails/SloganDescription.jsx

```diff
@@ -1,9 +1,8 @@
 /* eslint-disable react/prop-types */
```

Comment:
I think you should be able to delete this line now

Reply:
Forgot to delete, nice catch